ref/ams-configuration.yaml

```diff
@@ -43,14 +43,36 @@ all:
     description: >-
       APT mirror to use within the containers. By default,
       `http://archive.ubuntu.com` (amd64) or `http://ports.ubuntu.com`
+      (arm64) is used. This configuration item is deprecated since 1.20,
+      use `instance.apt_mirror` instead.
+  instance.apt_mirror:
```

Comment:
As mentioned above we should add a field to indicate the `container.` key variants of these are deprecated.

Reply:
@morphis Do you mean to say instead of adding as text as I have done, I should add a field that would display a separate column for the deprecated status? Currently, I have added that the `container.xxx` variants are deprecated in their respective descriptions. I could shift them to a separate column if that gives more clarity.

Reply:
Updated the `ams-configuration.yaml` to add a field for deprecated